File: go/hello.go
package main

import (
	"encoding/json"
	"net/http"
	"runtime"
	"strconv"
)

type MessageStruct struct {
	Message string
}

func hello(w http.ResponseWriter, r *http.Request) {
	w.Header().Set("Content-Type", "application/javascript")
	j, _ := json.Marshal(&MessageStruct{"Hello, world"})
	w.Header().Set("Content-Length", strconv.Itoa(len(j)))
	w.Write(j)
}

func main() {
	runtime.GOMAXPROCS(runtime.NumCPU())
	http.HandleFunc("/json", hello)
	http.ListenAndServe(":8080", nil)
}

File: uwsgi/hello.py
import ujson


def application(environ, start_response):
    response = {
      "message": "Hello, World!"
    }
    data = ujson.dumps(response)
    response_headers = [
        ('Content-type', 'application/json'),
        ('Content-Length', str(len(data)))
    ]
    start_response('200 OK', response_headers)
    return [data]
